pkg/kube_config_manager/kube_config_manager.go

```diff
@@ -72,6 +72,15 @@ func NewKubeConfigManager(ctx context.Context, bk backend.ConfigHandler, runtime
 	}
 }
 
+func (kcm *KubeConfigManager) IsModuleEnabled(moduleName string) bool {
+	moduleConfig, found := kcm.currentConfig.Modules[moduleName]
+	if !found {
+		return false
+	}
+
+	return *moduleConfig.IsEnabled
```

Comment:
`IsEnabled` could be nil. We have to add a check

Reply:
done